pandas-stubs/__init__.pyi

```diff
@@ -10,34 +10,69 @@ from .core.series import Series as Series
 from .core.indexes import Index as Index, MultiIndex as MultiIndex
 
 def concat(
-    dataframes: List[DataFrame],
-    axis: _AxisType = ...,
-    sort: Optional[bool] = ...,
-    ignore_index: bool = ...,
+    dataframes: List[DataFrame], axis: _AxisType = ..., sort: Optional[bool] = ..., ignore_index: bool = ...,
 ) -> DataFrame: ...
 def cut(arr: _np.ndarray, bins: int) -> Tuple[Union[Series, _np.ndarray], _np.ndarray]: ...
 def get_dummies(df: Union[DataFrame, Series], columns: Optional[_ListLike] = ...) -> DataFrame: ...
 def isnull(df: Union[DataFrame, Series]) -> _np.ndarray: ...
 @overload
 def merge(left: DataFrame, right: DataFrame, on: str = ...) -> DataFrame: ...
 @overload
-def merge(
-    left: DataFrame, right: DataFrame, left_on: str, right_on: str, how: str
-) -> DataFrame: ...
+def merge(left: DataFrame, right: DataFrame, left_on: str, right_on: str, how: str) -> DataFrame: ...
 @overload
-def merge(
-    left: DataFrame, right: DataFrame, left_on: List[str], right_on: List[str], how: str
-) -> DataFrame: ...
+def merge(left: DataFrame, right: DataFrame, left_on: List[str], right_on: List[str], how: str) -> DataFrame: ...
 def read_csv(
     filepath_or_buffer: Union[str, Path],
     sep: str = ...,
-    header: Optional[int] = ...,
+    delimiter: Optional[str] = ...,  # only an alias to sep
+    header: Optional[int, List[int], Literal["infer"]] = ...,
+    names: Optional[List[str]] = ...,
     index_col: Optional[Union[str, int, List[str], Tuple[str, ...], Sequence[int], bool]] = ...,
+    usecols: Optional[Union[List[str], List[int], Callable]] = ...,
+    squeeze: bool = ...,
+    prefix: Optional[str] = ...,
+    mangle_dupe_cols: bool = ...,
+    dtype: Optional[Union[Type, Dict[str, Type]]] = ...,
+    engine: Optional[Union[Literal["c"], Literal["python"]]] = ...,
+    converters: Dict[Union[str, int], Callable] = ...,
+    true_values: Optional[List] = ...,
+    false_values: Optional[List] = ...,
+    skipinitialspace: bool = ...,
+    skiprows: Optional[Union[int, _ListLike, Callable]] = ...,
+    skipfooter: int = ...,
+    nrows: Optional[int] = ...,
+    na_values: Optional[Union[str, List[str]]] = ...,
+    keep_default_na: bool = ...,
+    na_filter: bool = ...,
+    verbose: bool = ...,
+    skip_blank_line: bool = ...,
+    parse_dates: Union[bool, List[int], List[str], List[List[int]], Dict[str, List[int]]] = ...,
+    infer_datetime_format: bool = ...,
+    keep_date_col: bool = ...,
+    date_parser: Optional[Callable] = ...,
+    dayfirst: bool = ...,
+    cache_dates: bool = ...,
+    iterator: bool = ...,
+    chunksize: Optional[int] = ...,
+    compression: Optional[
+        Union[Literal["infer"], Literal["gzip"], Literal["bz3"], Literal["zip"], Literal["xz"]]
```

Comment:
A union of Literals can also be written as
```python
Literal["infer", "gzip", "bz3", "zip", "xz"]
```
makes it a bit shorter.

btw, can this list of literals also be used for `.to_csv()`?

Reply:
Yes, you're right! I was lazy and used _str instead